### ISSUE TITLE
Issue on page /notebooks/01 - Introduction.html

### ISSUE THREAD

**Issue by jeroenvanriel** [ML-course/master#36]
In section "Neural networks: evaluation and optimization", after "E.g. Gradient descent:", the formula is missing a partial derivative symbol and, more importantly, I guess there should be a minus in order to move towards the minimum.

**Comment by joaquinvanschoren** [comment]
Thanks!

I fixed the notebook and pushed it to GitHub. I will update the slides somewhat later this week or next week.
